CONTRIBUTING.md

```diff
@@ -461,6 +461,17 @@ output to a file, and grep through it.
 ./gradlew --debug 2>&1 > debugging-the-build.log
 ```
 
+### Getting More Output
+
+The Amplify Android library emits logs while it is running on a device
+or emulator. By default, debugging and verbose logs are not output.
+However, you can change the log treshold at runtime, by explicitly
+configuring a logging plugin:
+```kotlin
+Amplifly.addPlugin(AndroidLoggingPlugin(LogLevel.VERBOSE))
+// ... Add more plugins only *after* setting the log plugin.
```

Comment:
Hmm, what happens if I add the logging plugin last?  I didn't think order mattered.

Reply:
The other plugins may use logging while they are initializing themselves. The lifecycle of logging doesn't fit nicely into the "configure all at once" model.